website/src/content/docs/general/support-policy.md

```diff
@@ -1,41 +1,91 @@
 ---
-title: General Helm Support Policy
+title: Support Policy
+sidebar:
+  order: 2
 ---
 
-This manual contains general information on working with TrueCharts Charts using Helm directly.
-As well as when editing Helm Yaml for tools like FluxCD, KubeApps and Rancher
-
 :::tip
 
 Please, always remember to check the content specific to the chart.
 
 :::
 
-We at TrueCharts offer limited free support via a support-ticketing system on Discord.
-However the scope of this support is limited to two things:
+TrueCharts is a comprehensive project that focuses on providing opinionated Helm charts for applications to run on Kubernetes-based platforms.
+
+We also offer free support via a support-ticketing system on Discord.
+
+## Support Rules
 
-- ClusterTool Commands
-- The `values.yaml` of our Helm charts
+- If its covered by support as outlined below make a ticket in [⁠🎫・support](https://discord.com/channels/830763548678291466/936275413179723826).
+- [⁠🎫・support](https://discord.com/channels/830763548678291466/936275413179723826) is not rendered on GitHub or any other platform.
+- Bugs must be reported on GitHub, but only after confirmation by the staff via a [⁠🎫・support ticket](https://discord.com/channels/830763548678291466/936275413179723826)
+- Anything not covered by the below Support Policy is not eligble for a [⁠🎫・support ticket](https://discord.com/channels/830763548678291466/936275413179723826). If you need assitance outside the scuoe of [⁠🎫・support](https://discord.com/channels/830763548678291466/936275413179723826) make a thread in an appropiate [Discord Channel](https://discord.gg/tVsPTHWTtr).
 
 ## Expected Prior Knowledge
 
-- How to use a shell/console/terminal
-- Basic shell commands (`cp`, `ls`, `etc` and if you think `etc` is a command here, you'll have a problem)
-- Basic `kubectl` use (get pods, get svc, describe something etc)
-- Basic knowhow of Talos when you use it (things like: what is "Apply", what is a "machineconfig", what does "upgrade" do vs `k8s-upgrade`)
-- Basic Kubernetes knowhow (such as what is a "worker" node vs a "controlplane" node, what's "etcd", what's a "pod" vs a "container", etc.)
+- How to use a shell/console/terminal and basic commands (ie. `cp`, `ls`, `etc`).
+- Basic Kubernetes know how (such as what is a "worker" node vs a "controlplane" node, what's "etcd", what's a "pod" vs a "container", etc).
+
+## General Support Policy
+
+- We help out where we can, as long as you tried to do it yourself *first*. We do not provide a walkthrough service.
+- Our staff will try to help out with providing instructions on how to use `kubectl` and other `*ctl` tools where needed.
+- Please be aware that the code of conduct also applies to all support.
+- We do not support what we do not build.
```

Comment:
```suggestion
- We do not support what we do not build.
```

lets get rid of this, I think this new policy is more clear anyway and its not 100% true either